### PR TITLE
(FM-7747) IOS Network DNS Created

### PR DESCRIPTION
**Pull request by david22swan**
- ip_domain_lookup attribute added

### CODE REVIEW

File: lib/puppet/provider/ios_network_dns/command.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+attributes:
+  #USAA
```
Comment (DavidS) [inline lib/puppet/provider/ios_network_dns/command.yaml]: this shouldn't be here.

Comment (david22swan) [reply lib/puppet/provider/ios_network_dns/command.yaml]: removed

**Comment by shermdog** [comment]
This may be better suited for netdev as both IOS and NXOS support this field.

File: README.md [hunk 2]
```diff
@@ -1042,6 +1043,27 @@ ID of the stp global config. Valid values are default.
...
+Data type: `Optional[Boolean]`
+
+Sets whether the Domain Name Server (DNS) lookup feature should be enabled, by default this should return true.
+
+Examples:
+```
+ip_domain_lookup => false
+```
+
+See `network_dns` for other availible fields.
```
Comment (da-ar) [inline README.md]: spelling: available

File: README.md [hunk 3]
```diff
@@ -1042,6 +1043,27 @@ ID of the stp global config. Valid values are default.
...
+
+#### Properties
+
+The following properties are availible in the `ios_network_dns` type.
+
+##### ip_domain_lookup
+
+Data type: `Optional[Boolean]`
+
+Sets whether the Domain Name Server (DNS) lookup feature should be enabled, by default this should return true.
```
Comment (da-ar) [inline README.md]: As this is an Optional value, this `by default this should return true.` does not make sense.
The example can also be removed as it doesn't demonstrate anything complex.

File: README.md (hunk 2 above)
Comment (david22swan) [reply README.md]: corrected

File: README.md (hunk 3 above)
Comment (david22swan) [reply README.md]: have updated the wording and removed the example

File: README.md (hunk 2 above)
Comment (da-ar) [reply README.md]: Not corrected 😛

File: README.md [hunk 4]
```diff
@@ -1141,6 +1142,22 @@ ID of the stp global config. Valid values are default.
...
+
+#### Properties
+
+The following properties are available in the `ios_network_dns` type.
+
+##### ip_domain_lookup
+
+Data type: `Optional[Boolean]`
+
+Sets whether the Domain Name Server (DNS) lookup feature should be enabled, this feature is enabled by default on most cisco machines.
```
Comment (da-ar) [inline README.md]: I think this should just read "Sets whether the Domain Name Server (DNS) lookup feature should be enabled"

Comment (david22swan) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (david22swan) [reply README.md]: Corrected for real

**Pull request merged**